requests-for-comments/2023-17-produchog.md

```diff
@@ -0,0 +1,92 @@
+# RFC: The future of HogMail is... ProductHog
+
+HogMail was launched in April as a mostly PostHog orientated news drop. It’s since evolved into a more product growth-themed newsletter hybrid. We send it every two weeks – this won’t change.
+
+This RFC sets out some ideas and discussion points for taking HogMail to the next level by treating it as a distinct brand and product, created by PostHog.
+
+Most recent issue for context + feedback: [https://mailchi.mp/posthog/hogmail-15794324](https://mailchi.mp/posthog/hogmail-15794324)
+
+## Where we are now
+
+- Our subscriber list has grown 49% since April – 662 to 987 as of writing.
+- Open rates have increased from ~43% at launch to ~47% to 50% now. Based on my research, this is above average but not market-leading for a newsletter of our type.
+- Each issue is split into three sections: new PostHog articles, curated articles with commentary, misc. interesting reads.
+- Most signups come via CTAs on the website and promotion in The Array
+- At the current rate of growth, we’d reach 1,500 subscribers by the end of 2023
+
+## Where we want to be
+
+- 3,000 subs by end of 2023 – a 200% increase on today
+- People sharing and recommending newsletter on social media
+- Be a newsletter that’s perceived as a desirable source of promotion for content creators
```

Comment:
Agreed, I think there is no reason we shouldn't be in the same league as the newsletters we sponsor (or better).

Reply:
This is a good way to frame this.